docs/services/servers.md

```diff
@@ -1,18 +1,122 @@
 # Servers
 
-Redbrick provides two main servers (Azazel and Pygmalion) for it's members to use for various use cases, for example running programs.
+Redbrick provides two main servers (Azazel and Pygmalion) for it's members to use for various use cases, for example running applications or user programs.
 
-## Logging in to Redbrick Servers
+## Entrypoints
 
 The main login server used in Redbrick is Azazel. You may also log in to Pygmalion if you wish at `pyg.redbrick.dcu.ie` 
 
-You can log in using ssh in your terminal/command prompt application of choice with your Redbrick username and password like so:
+**2 Factor Authentication is required to log in to Redbrick servers.** This is done via an SSH key and your Redbrick username/password combination. For more information on how to create an SSH key, and configure your account for 2FA, please read below.
+
+## Logging in
+
+You've set up 2FA on your account with an SSH key, right? [_If not, you really have to, I'm sorry._](#setting-up-an-ssh-key)
+
+You can log in using SSH in your command prompt or terminal application of choice with your Redbrick username and password like so:
 
 ```bash
-ssh YOUR_USERNAME@redbrick.dcu.ie
+ssh YOUR_USERNAME@redbrick.dcu.ie -i /home/bob/.ssh/id_ed25519
```

Comment:
Change this to maybe just `<ssh key location>`

Reply:
Maybe also consider mentioning adding a host to `~/.ssh/config`